man/systemd.exec.xml

```diff
@@ -89,6 +89,17 @@
 
     <variablelist class='unit-directives'>
 
+      <varlistentry>
+        <term><varname>BinaryDirectory=</varname></term>
+
+        <listitem><para>Takes a directory path relative to which the executable used by
+        the Exec*= (e.g. ExecStart=, ExecStop=, etc.) properties can be specified.
+        The name of the executable has to be non standard.
```

Comment:
```suggestion
        The name of the executable has to be nonstandard.
```
but I think this sentence is a bit confusing. I think you mean, that the binary has to be in a directory that normally is not in `PATH`, right?

Reply:
What I noticed when I was testing was that if a command such as ```echo`` was passed it would replace with "/bin/echo".

Reply:
Yes, exactly.